metadata.rb

```diff
@@ -7,9 +7,9 @@
 source_url       'https://github.com/dnsimple/chef-pdns'
 issues_url       'https://github.com/dnsimple/chef-pdns/issues'
 
-chef_version '>= 15'
+chef_version '>= 18'
```

Comment:
This is a major change, is 18 needed, or can we keep this?
At minimum chef 17 is still not EoL until 2024
https://docs.chef.io/versions/